scripts/istat/geos/preprocess.py

```diff
@@ -0,0 +1,96 @@
+# Lint as python3
+"""Preprocess the ISTAT geo code to NUTS code for importing into Data Commons"""
+
+import pandas as pd
+
+FILE_PATH_it = "./Elenco-codici-statistici-e-denominazioni-al-01_01_2020_it.csv"
+FILE_PATH_en = "./Elenco-codici-statistici-e-denominazioni-al-01_01_2020_en.csv"
+
+
+def translate():
+  # tranlate the column names to English based on Google translate
+  data = pd.read_csv(FILE_PATH_it, sep = ';', encoding = 'cp1252')
```

Comment:
Wow.. cp1252.. I have never seen that. Just curious, how did you figure out what to encode?

Reply:
I used a python library "chardet" to detect the encoding.